eno-core/src/main/java/fr/insee/eno/core/processing/out/steps/lunatic/LunaticAddMissingVariables.java

```diff
@@ -123,6 +124,7 @@ private List<String> getMissingBlockNames(ComponentType component) {
                             .toList();
             case ROSTER_FOR_LOOP ->
                     names = ((RosterForLoop)component).getComponents().stream()
+                            .filter(Objects::nonNull)
```

Comment:
Maybe we should remove this filter: there should be no nulls in this list -> if a null here makes the program crash, figure out why there is a null here (and fix the piece of program that puts nulls)

Reply:
https://github.com/InseeFr/Eno/pull/633/commits/2e565c4bc4f5fa574aca1dfa6210b634ca7534cf